src/types/poll.rs

```diff
@@ -47,3 +48,14 @@ pub struct PollOption {
     /// Number of users that voted for this option.
     pub voter_count: i32,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn deserialize() {
+        let data = r#"{"allows_multiple_answers":false,"id":"5377643193141559299","is_anonymous":true,"is_closed":false,"options":[{"text":"1","voter_count":1},{"text":"2","voter_count":0},{"text":"3","voter_count":0},{"text":"4","voter_count":0},{"text":"5","voter_count":0}],"question":"Rate me from 1 to 5.","total_voter_count":1,"type":"regular"}"#;
```

Comment:
Could you please format this JSON? Here, for example: http://jsonprettify.com/.

Reply:
Formatted.